src/server/gc_worker/compaction_filter.rs

```diff
@@ -235,10 +242,12 @@ impl CompactionFilterFactory for WriteCompactionFilterFactory {
         }
         drop(gc_context_option);
 
-        GC_COMPACTION_FILTER_PERFORM.inc();
+        GC_COMPACTION_FILTER_PERFORM
+            .with_label_values(&["txn"])
```

Comment:
Suggest to define constants for labels `"txn"` & `"raw"` to avoid typos.

Reply:
Add the constants in gc_worker.rs ,it's ok?